src/raqm.c

```diff
@@ -1009,6 +1009,69 @@ raqm_get_glyphs (raqm_t *rq,
   return rq->glyphs;
 }
 
+/**
+ * raqm_get_par_resolved_direction:
+ * @rq: a #raqm_t.
+ *
+ * Gets the resolved direction of the paragraph;
+ *
+ * Return value:
+ * The #raqm_direction_t specifying the resolved direction of text,
+ * or #RAQM_DIRECTION_DEFAULT.
+ *
+ * Since: 0.8
+ */
+RAQM_API raqm_direction_t
+raqm_get_par_resolved_direction (raqm_t *rq)
+{
+  if (!rq)
+    return RAQM_DIRECTION_DEFAULT;
+
+  if (rq->base_dir == RAQM_DIRECTION_TTB)
+    return RAQM_DIRECTION_TTB;
```

Comment:
We probably should set `rq->resolved_dir` to `RAQM_DIRECTION_TTB` in this case instead of the current `RAQM_DIRECTION_LTR` which is no longer needed (we were doing this when we were passing TTB text to FriBiDi, but we no longer do this).

Reply:
Done